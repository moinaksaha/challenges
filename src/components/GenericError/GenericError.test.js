import React from 'react'
import { shallow } from 'enzyme';
import Genericerror from './index.js'
import toJson from 'enzyme-to-json';

describe('Component: <Genericerror />', () => {

  test('Render Status: Success', () => {
    const wrapper = shallow(<Genericerror />);
    expect(wrapper.exists()).toBe(true);
  })

  test('Snapshot: Enzyme', () => {
    const tree = shallow(<Genericerror />)
    expect(toJson(tree)).toMatchSnapshot()
  })
  
})