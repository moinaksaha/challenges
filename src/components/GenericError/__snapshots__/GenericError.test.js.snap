// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Genericerror /> Snapshot: Enzyme 1`] = `
<styled.div>
  <h3>
    Oops!
  </h3>
  <p>
    Something went wrong, Please try again later
  </p>
</styled.div>
`;
